README.md

```diff
@@ -427,6 +427,74 @@ You can specify placeholder text for search- and select-type filters by setting
 
 If the filter is search-type, the `placeholder` attribute of the `<input>` element will be set to the placeholder value. If the filter is select-type, the text of the first `<option>` of the `<select>` element will be set to the placeholder value.
 
+#### Custom Filter Components
+
+You may want to render something other than a text box or select list for the filter. To do so, specify the component to render by providing
+the name in the `component` property of the `filter` object.
+
+For example, given that a component named "checkbox-filter" exists:
+
+```javascript
+[
+  {
+    key: 'name',
+    header: 'Name',
+    filter: {
+      component: 'checkbox-filter'
+    }
+  }
+]
+```
+
+Note that the `component` and `type` properties are mutually exclusive.
```

Comment:
Does this mean that if `component` is present, then `type` will be ignored? Or vice versa? Not super important (because it's invalid configuration), but I'm curious about what happens if you specify both. Might be a good test case.